open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb

```diff
@@ -57,7 +57,7 @@
    "id": "f5ee754a-e5d9-4532-86d5-1433722be122",
```

Comment:
<p>I think a main heading should be added, and then this heading should be changed to a secondary heading. </p><p>And the same operation should be done for all the following primary headings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>